wurst/event/DamageEvent.wurst

```diff
@@ -0,0 +1,238 @@
+package DamageEvent
+
+import ClosureEvents
+import UnitIndexer
+
+
+/* CONFIGURATION */
+
+@configurable constant DETECT_NATIVE_ABILITIES = false // Not implemented
+
+@configurable public constant DAMAGE_ELEMENT_PHYSICAL = new DamageElement("Physical")
+
+
+/* DAMAGE TYPE */
```

Comment:
Please add actual hotdoc `/** */` to these top level declarations, explaining what they do / are used for.
Right now these comments seem a bit superfluous.